manifests/foreman.pp

```diff
@@ -55,12 +55,11 @@
     }
 
     file { $ssl_ca_cert:
-      ensure  => file,
-      source  => $server_ca,
-      owner   => 'root',
-      group   => $group,
-      mode    => '0440',
-      require => File[$server_ca],
```

Comment:
Isn't this still required? Or does Puppet automatically create that dependency since it's specified as a source?

Reply:
From my experience and testing, yes, Puppet handles this since we set the source.

Reply:
This `require` is not strictly required, but not because it's autorequired via the `source` parameter. It's because that's a resource contained in the `certs::ca` class that you require in line #44.